src/modules/SdnDiag.Health.psm1

```diff
@@ -2402,6 +2403,33 @@ function Test-SdnVfpEnabledVMSwitch {
     return $sdnHealthTest
 }
 
+function Test-SdnVfpEnabledVMSwitch {
+    <#
+        .SYNOPSIS
+            Enumerates the VMSwitches on the system and validates that only one VMSwitch is configured with VFP.
+    #>
+
+    [CmdletBinding()]
+    param()
+
+    Confirm-IsServer
+    $sdnHealthTest = New-SdnHealthTest
+
+    try {
+        # if there is more than one VMSwitch configured with VFP, this is a failure
+        $vmSwitches = Get-SdnVMSwitch -VfpEnabled
+        if ($vmSwitches.Count -eq 0 -or $null -eq $vmSwitches) {
```

Comment:
doesn't this conditional only test if there are no vswitches with vfp enabled (e.g., if there are 2 vswitches, it won't fail)?

Reply:
So in this test specifically, I am only ensuring that if no switches returned, to fail the test. In situation where there might be two VMSwitches, with VFPEnabled, it would flag via `Test-SdnVfpEnabledVMSwitchMultiple` but not this one.

Reply:
@andrwli fixed the comments within the tests to make this more clear